.github/workflows/base-release.yaml

```
@@ -92,7 +92,7 @@ jobs:
 
       - uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5 # v5.5.0
         with:
-          go-version: "~1.24"
+          go-version: "~1.25.0"
```

reverting this for now
```suggestion
          go-version: "~1.24"
```